CoroutineTemplate/build.gradle

```diff
@@ -0,0 +1,77 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext {
+        build_gradle_version = '4.1.1'
+
+        android_version_code = 9
+        android_version_name = "0.7.0"
```

Comment:
which version should we start?

Reply:
Oh, I think we can start with 0.1.0 🙏  What do you think? @Tuubz @chornerman